cbmgmtx/errors.go

```diff
@@ -42,3 +43,17 @@ func (e contextualError) Error() string {
 func (e contextualError) Unwrap() error {
 	return e.Cause
 }
+
+type ServerInvalidArgError struct {
+	cause    error
```

Comment:
I don't think this needs a cause? Unwrap should just directly always return ErrServerInvalidArg.

Reply:
Good point, done now